mobile/lib/features/trade/channel_creation_flow/channel_funding_screen.dart

```diff
@@ -0,0 +1,452 @@
+                  child: Selector<FundingChannelChangeNotifier, FundingChannelTaskStatus?>(
+                      selector: (_, provider) {
+                    return provider.status;
+                  }, builder: (BuildContext context, value, Widget? child) {
+                    if (value case FundingChannelTaskStatus.orderCreated) {
+                      return ElevatedButton(
+                        onPressed: () {
+                          GoRouter.of(context).go(TradeScreen.route);
+                        },
+                        style: ElevatedButton.styleFrom(
+                            minimumSize: const Size.fromHeight(50),
+                            backgroundColor: tenTenOnePurple),
+                        child: const Text(
+                          "Home",
+                          style: TextStyle(color: Colors.white),
+                        ),
+                      );
+                    } else {
+                      return buildButtonRow();
+                    }
+                  })),
+            ],
+          ),
+        ),
+      ),
+    );
+  }
+
+  Row buildButtonRow() {
+    return Row(
+      children: [
+        Expanded(
+          child: ClickableBox(
+            text: "Unified",
+            image: const Icon(BitcoinIcons.bitcoin_circle_outline),
+            isSelected: selectedBox == FundinType.unified,
+            onTap: () {
+              setState(() {
+                selectedBox = FundinType.unified;
+              });
+            },
+          ),
+        ),
+        Expanded(
+          child: ClickableBox(
+            text: "Lightning",
+            image: const Icon(BitcoinIcons.lightning_outline),
+            isSelected: selectedBox == FundinType.lightning,
+            onTap: () {
+              setState(() {
+                selectedBox = FundinType.lightning;
+              });
+            },
+          ),
+        ),
+        Expanded(
+          child: ClickableBox(
+            text: "On-chain",
+            image: const Icon(BitcoinIcons.link_outline),
+            isSelected: selectedBox == FundinType.onchain,
+            onTap: () {
+              setState(() {
+                selectedBox = FundinType.onchain;
+              });
+            },
+          ),
+        ),
+        Expanded(
+          child: ClickableBox(
+            text: "External",
+            image: const Icon(BitcoinIcons.wallet),
+            isSelected: selectedBox == FundinType.external,
+            onTap: () {
+              setState(() {
+                selectedBox = FundinType.external;
+              });
+            },
+          ),
+        )
+      ],
+    );
+  }
+
+  Column buildInfoBox(FundingChannelTaskStatus? value, FundinType selectedBox) {
+    String transactionStatusText = "Waiting for payment...";
+    String transactionStatusInformationText =
+        "Please wait. If you leave now, your position won’t be opened when the funds arrive.";
```

Comment:
🤔 if the user would want to switch to another app to send the onchain funds this message might be misleading.

Reply:
Could be. Do you have a proposal to change this?